.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -0,0 +1,62 @@
+## Test plan
+
```

Comment:
Should this issue template's goal be part of the user story issue instead?  Why have a separate issue?  A user story is not done until it is QA'd, and there's a column for that.